test/config-luatest/rpc_test.lua

```diff
@@ -780,3 +803,104 @@ g.test_call_mode = function(g)
     g.server_3:exec(check)
     g.server_4:exec(check)
 end
+
+g.test_call_pick_any = function(g)
+    local dir = treegen.prepare_directory(g, {}, {})
+    local config = [[
+    credentials:
+      users:
+        guest:
+          roles: [super]
+        myuser:
+          password: "secret"
+          roles: [replication]
+          privileges:
+          - permissions: [execute]
+            universe: true
+
+    roles: [one]
+
+    iproto:
+      listen:
+        - uri: 'unix/:./{{ instance_name }}.iproto'
+      advertise:
+        peer:
+          login: 'myuser'
+
+    groups:
+      group-001:
+        replicasets:
+          replicaset-001:
+            instances:
+              instance-001:
+                database:
+                  mode: rw
+              instance-003: {}
+          replicaset-002:
+            instances:
+              instance-002:
+                iproto:
+                  listen:
+                    - uri: 9.0.0.0:8888
```

Comment:
[Bad](https://github.com/tarantool/tarantool/pull/10287#discussion_r1773819957) idea.

Reply:
Fixed this problems by starting stub tcp servers. Thank you.